src/server/game/Entities/AreaTrigger/AreaTriggerTemplate.h

```diff
@@ -71,6 +72,16 @@ enum AreaTriggerActionUserTypes
     AREATRIGGER_ACTION_USER_MAX    = 6
 };
 
+struct AreaTriggerServerPosition
+{
+    uint32 Id;
+    bool IsServer;
+    WorldLocation Location;
```

Comment:
A few more fields are needed here (and in database)
* uint64 SpawnId; // guid, PK (to support multiple spawns with the same template/script - imaginary scenario: a few creatures with areatrigger around each of them doing the same thing for "go near one of them" type of script)
* uint8 PhaseUseFlags; // phasing requirement - for example to only do thing when you are phased on a quest
* uint32 PhaseId;
* uint32 PhaseGroup;